### PR TITLE
Update common references to v4.6 in TestWorkspace

### PR DESCRIPTION
**Pull request by cbo425**

### CODE REVIEW

**Comment by cbo425** [comment]
@dotnet/roslyn-infrastructure  2 VSI tests failed in https://ci.dot.net/job/dotnet_roslyn/job/master/job/windows_release_vs-integration_prtest/1820/

**Comment by cbo425** [comment]
retest windows_release_vs-integration_prtest please

**Comment by cbo425** [comment]
tagging @dotnet/roslyn-ide for review

File: src/EditorFeatures/Test/MetadataAsSource/MetadataAsSourceTests.VisualBasic.cs [hunk 1]
```diff
@@ -31,15 +31,16 @@ End Class
 End Module");
             }
 
+        // This test depends on the version of mscorlib used by the TestWorkspace and may
```
Comment (dpoeschl) [inline src/EditorFeatures/Test/MetadataAsSource/MetadataAsSourceTests.VisualBasic.cs]: This comment should be indented to the same level as the method

**Pull request merged**